packages/fx-core/resource/package.nls.json

```diff
@@ -1078,8 +1078,8 @@
   "driver.aadApp.description.create": "Create an Azure Active Directory app to authenticate users",
   "driver.aadApp.description.update": "Apply Azure Active Directory app manifest to an existing app",
   "driver.aadApp.error.invalidParameter": "Following parameter is missing or invalid for %s action: %s.",
-  "driver.aadApp.error.missingEnv": "%s Environment variable %s is not set.",
-  "driver.aadApp.error.missingEnvInFile": "%s Environment variable %s referenced in %s have no values.",
+  "driver.aadApp.error.missingEnv": "Environment variable %s is not set.",
+  "driver.aadApp.error.missingEnvInFile": "Environment variable %s referenced in %s have no values.",
```

Comment:
have -> has
values -> value.

Or bring all to plural form to be consistent.

Reply:
changed